### PR TITLE
CF-330 Remove failed volumes

### PR DESCRIPTION
**Pull request by roman-verchikov**
Modifies logic to correctly remove volumes which failed to migrate for
any reason.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferrylib/os/actions/migrate_volumes.py [hunk 1]
```diff
@@ -134,57 +134,81 @@ def migrate_volume(self, src_volume):
...
+                   "you migrated tenants.").format(
+                src_volume['project_id'])
+            LOG.warning(msg)
+            raise exception.TenantNotPresentInDestination(msg)
+
+        LOG.info("Creating volume of size %sG in tenant %s in destination",
+                 src_volume['size'], dst_tenant.name)
+        dst_volume = dst_cinder.create_volume_from_volume(src_volume,
+                                                          dst_tenant.id)
+        LOG.info("Volume created: %s", volume_name(dst_volume))
```
Comment (SVilgelm) [inline cloudferrylib/os/actions/migrate_volumes.py]: please add the id of volume into log message

Comment (roman-verchikov) [inline cloudferrylib/os/actions/migrate_volumes.py]: It's there already, see `volume_name` implementation

Comment (SVilgelm) [inline cloudferrylib/os/actions/migrate_volumes.py]: ok, thanks

**Comment by roman-verchikov** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**